components/brave_shields/browser/brave_shields_web_contents_observer_android.cc

```diff
@@ -0,0 +1,36 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_shields/browser/brave_shields_web_contents_observer.h"
+
+#include <map>
```

Comment:
some of these includes don't appear to be used

Reply:
yeah, it looks like I just copied includes from a base observer. Corrected.